parts_list/master_parts_list_raw.csv

```diff
@@ -142,3 +142,5 @@ GoBilda 23mm threaded plate,S46,2803-0023-0023,GoBilda,https://www.gobilda.com/2
 2-post 43-32mm drive motor clamping mount,S47,1401-0043-0032,GoBilda,https://www.gobilda.com/1401-series-2-side-2-post-clamping-mount-43mm-width-32mm-bore/,1,6,6,$5.99,$35.94,Wheel Assembly
 GoBilda Pattern Adaptor,S48,1206-0016-0004,GoBilda,https://www.gobilda.com/1206-series-pattern-adaptor-16-4/,1,12,12,$4.99,$59.88,Wheel Assembly
 6mm Bore Clamping D-Hub 0.770in,S49,545618,ServoCity,https://www.servocity.com/6mm-bore-clamping-d-hub-tapped-0-770-pattern/,1,6,6,$7.99,$47.94,Wheel Assembly
+3D Printed PVC Clamping Hub (bottom),S50,,Makexyz,https://www.makexyz.com/,1,1,1,$12.90,$12.90,Body, Head Assembly
+9 x 12 Bottom Plate,S51,,Sculpteo,https://www.sculpteo.com/,1,1,1,$11.43,$11.43,
```

Comment:
These lines need to be removed, and moved to https://github.com/nasa-jpl/open-source-rover/pull/259